src/Control/Effect/Labelled.hs

```diff
@@ -70,6 +77,35 @@ instance Algebra (eff :+: sig) (sub m) => Algebra (Labelled label eff :+: sig) (
     R sig -> Labelled . alg (runLabelled . hdl) (R sig)
   {-# INLINE alg #-}
 
+type HasLabelledLift n sig m = HasLabelled Lift (Lift n) sig m
+
+newtype LabelledLift (label :: k) (sub :: (Type -> Type) -> (Type -> Type)) m a = LabelledLift (m a)
+  deriving
+    ( Alternative
+    , Applicative
+    , Functor
+    , Monad
+    , Fail.MonadFail
+    , MonadFix -- ^ @since 1.1.1
+    , MonadIO
+    , MonadPlus
+    )
+
+runLabelledLift :: forall sub m a . LabelledLift Lift sub m a -> m a
+runLabelledLift (LabelledLift l) = l
+{-# INLINE runLabelledLift #-}
+
+instance Algebra eff m => Algebra (Labelled Lift eff) (LabelledLift Lift sub m ) where
+    alg hdl (Labelled sub) = LabelledLift . alg (runLabelledLift . hdl) sub
+    {-# INLINE alg #-}
+
+lift
```

Comment:
There seems to me a slight possibility that this could create name collisions if people imported both `Effect.Lift` and `Labelled`, but I think the chance of things actually breaking is minimal. I’ll reach out to some users and see if they use `Labelled` and `lift` in conjunction.

Reply:
I'm more concerned about the conflict with `MonadTrans`. Thoughts, @patrickt?

Reply:
`Lift n` corresponds to the bottom monad `n` of the monad stack, which is unique. So `Lift n` and `HasLabelledLift n` should not exist at the same time, but choose one or the other. So there is no conflict between them.

Reply:
Is it better to use sendM instead of lift?